### PR TITLE
aws: Attach security group to NLBs for kops-controller

### PR DESCRIPTION
**Pull request by hakman**
This was broken in #15993

/cc @johngmyers @rifelpet

### CODE REVIEW

**Comment by johngmyers** [comment]
I think we want to add port 3988 to the appropriate list in AWSModelContext.GetSecurityGroups()

**Comment by hakman** [comment]
> I think we want to add port 3988 to the appropriate list in AWSModelContext.GetSecurityGroups()

Done

File: pkg/model/awsmodel/api_loadbalancer.go [hunk 1]
```diff
@@ -517,6 +535,17 @@ func (b *APILoadBalancerBuilder) Build(c *fi.CloudupModelBuilderContext) error {
 				SourceGroup:   masterGroup.Task,
 				ToPort:        fi.PtrTo(int64(4)),
 			})
+			if b.Cluster.UsesNoneDNS() {
+				c.AddTask(&awstasks.SecurityGroupRule{
+					Name:          fi.PtrTo(fmt.Sprintf("kops-controller-elb-to-master%s", suffix)),
```
Comment (johngmyers) [inline pkg/model/awsmodel/api_loadbalancer.go:540]:
```suggestion
					Name:          fi.PtrTo(fmt.Sprintf("kops-controller-elb-to-cp%s", suffix)),
```

**Review by johngmyers — CHANGES_REQUESTED** [review]

**Review by upodroid — APPROVED** [review]
LGTM

**Pull request merged**